pkg/util/pluginversion/pluginversion.go

```diff
@@ -0,0 +1,40 @@
+package pluginversion
+
+import "fmt"
+
+// Version represents the plugin version
+type Version struct {
+	Major  int
+	Minor  int
+	Latest bool
```

Comment:
yikes - when do we need this case?

Reply:
Just trying to be complete, i'll remove.